content/tracing/visualization/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Getting started with APM
+kind: documentation
+further_reading:
+- link: "/tracing/setup/"
+  tag: "Documentation"
+  text: Learn how to setup APM tracing with your application
+- link: "/tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: Discover the list of services reporting to Datadog
+- link: "/tracing/visualization/service"
+  tag: "Documentation"
+  text: Learn more about services in Datadog
+- link: "/tracing/visualization/resource"
+  tag: "Documentation"
+  text: Dive into your resource performances and traces
+- link: "/tracing/visualization/trace"
+  tag: "Documentation"
+  text: Understand how to read a Datadog Trace
+---
+
+APM collects metrics on your app’s performance at four levels of granularity: _services_, _resources_, _trace_, and _span_ level.
+
+## Services
+
+**A service is a set of processes that do the same job.**  
+For instance, a simple web application may consist of two services: 
+
+* A single `webapp` service and a single `database` service
```

Comment:
This bullet point doesn't have a period, but the two following it do.

Reply:
👌